src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
```

Comment:
`auto` is the default since it the current time scale behavior, changing it will be a breaking change. `auto` generates linear ticks and in this case `series` or `linear` modes have exactly the same result. Now, let's say you change source for `labels`, as a user I would expect to have the `series` alignment of ticks by default. That why I picked `series` and `auto` by default.